harvester/attachedcontainer.go

```diff
@@ -111,8 +111,13 @@ func (a *AttachedContainer) createContainer() {
 	schedulerAliasHostname := "scheduler"
 	// Add an aliass for the scheduler to each harvester
 	links := []string{fmt.Sprintf("%s:%s", a.schedulerID, schedulerAliasHostname)}
+	labels := map[string]string{"lumogon_attached_container": "true",
```

Comment:
@jfryman / @garethr thinking I should probably use a report specific unique id here and only delete harvesters with that matching ID - just in case there are concurrent harvestings going on?

Reply:
Agreed, as I was working through the changes I was thinking about race conditions.

Reply:
I'm generating the reportID when the scheduler is initialised and pushing that out through the system, it had previously been generated at storage - makes more sense this way so a double win.